src/overlays/actors/ovl_En_Md/z_en_md.h

```diff
@@ -35,12 +35,12 @@ typedef struct EnMd {
     /* 0x0190 */ EnMdActionFunc actionFunc;
     /* 0x0194 */ ColliderCylinder collider;
     /* 0x01E0 */ NpcInteractInfo interactInfo;
-    /* 0x0208 */ u8 unk_208;
-    /* 0x0209 */ u8 unk_209;
-    /* 0x020A */ u8 unk_20A;
-    /* 0x020B */ u8 unk_20B;
+    /* 0x0208 */ u8 messageEntry; // tracks message state changes, in place of `.actor.textId`
+    /* 0x0209 */ u8 messageState; // last known result of `Message_GetState`
+    /* 0x020A */ u8 animEntry;    // each one changes animation info and waits
+    /* 0x020B */ u8 animState;    // defines a sequence of animation entries
```

Comment:
(also as a minor nit, we;ve been preferring to have comments come immediately after a field, instead of trying to pad them out to line them up)
as in,
`u8 animEntry; // each one changes animation info and waits`
instead of 
`u8 animEntry;    // each one changes animation info and waits`

Reply:
Yeah `animSequenceEntry` and `animSequence` seems good to me, it makes the relationship more clearer

`format.py` currently only runs clang-format on `src/**/*.c`, and only does whitespace cleanups on `assets/**/*.xml`, `include/**/*.h`, and `src/**/*.h`. I think the plan is to auto-format headers eventually but I'm not sure what's going on with that. Personally I have my editor's format-on-save turned off for this project, and I either run format.py manually or just let Jenkins fix it, but that is kinda annoying

Reply:
Dont think we've ever considered formatting headers. We commonly break from the mold on the formatting rules in headers for the sake of custom formatting for documentation

Reply:
should we go and drop `.clang-format`
```
DisableFormat: true
```
into the `./include` directory

and / or
```
// clang-format off
// clang-format on
```
for the headers in `./src`

?

Reply:
hm it would be nice to do something like that, although there are quite a lot of headers in `src` so idk about adding `clang-format off` to all of them. (If you do find a goood solution, it should be a separate PR though pls)